config.yml.erb

```diff
@@ -387,11 +387,8 @@ optimize_webpack_assets: true
 
 optimize_rails_assets: true
 
-# Allows your local server to render non-English locales, defaults to false in development.
-# If false, choosing a locale other than English will have no effect.
-# Note: You may need to be in an incognito window to see changes
-# due to a difference in the language cookie between production and localhost.
-load_locales:
+# Allows your local server to lazy load locales.
```

Comment:
might be worth explaining here that this setting is primarily intended for use in the local development environment to speed up dev iteration time, and is not intended for production use.